src/runtime/auto-prompt-manager-test.js

```diff
@@ -2162,6 +2193,32 @@ describes.realWin('AutoPromptManager', (env) => {
     await tick(2);
   }
 
+  function resolveArticleExpectationOnce(
+    articleExpectation,
+    actionTypes,
+    experimentFlags,
+    numReadsBetweenPrompts = 0
+  ) {
+    const actions =
+      actionTypes.map((actionType) => {
```

Comment:
Nit: This could be a one liner
```suggestion
      actionTypes.map((actionType) => ({ type: actionType })
```